core/chains/evm/client/node.go

```diff
@@ -454,7 +454,7 @@ func (n *node) BatchCallContext(ctx context.Context, b []rpc.BatchElem) error {
 	defer cancel()
 	lggr := n.newRqLggr(switching(n)).With("nBatchElems", len(b), "batchElems", b)
 
-	lggr.Debug("RPC call: evmclient.Client#BatchCallContext")
+	lggr.Trace("RPC call: evmclient.Client#BatchCallContext")
```

Comment:
Sometimes these can be really useful, ideally I'd prefer to see us implement https://smartcontract-it.atlassian.net/browse/BCI-1411

Reply:
Do you mean everything? Or just names w/o data?